src/VisualStudio/Core/Impl/RoslynVisualStudioWorkspace.cs

```diff
@@ -29,16 +30,19 @@ namespace Microsoft.VisualStudio.LanguageServices
     internal class RoslynVisualStudioWorkspace : VisualStudioWorkspaceImpl
     {
         private readonly IEnumerable<Lazy<IStreamingFindUsagesPresenter>> _streamingPresenters;
+        private readonly IEnumerable<Lazy<IExternalNavigationService>> _symbolicNavigationServices;
```

Comment:
Love the new name! We'll want to rename the fields/locals to match though.

Reply:
Woops, how embarassing :). Should be done now.